### ISSUE TITLE
Red colored text looks orange

### ISSUE THREAD

**Issue by alessandromrc** [anyproto/anytype-ts#1097]
### Have you read a contributing guide?

- [X] I have read CONTRIBUTING.md
- [X] I have searched the existing issues and didn't find any that were similar
- [X] I have considered creating a pull request with fixes instead of a bug report and want to proceed

### Current Behavior

As you can see here:

![image](https://github.com/user-attachments/assets/5e3941f5-c2f4-4fa3-a9f2-7562223f081d)

The text looks orange instead of red, I think the hex color is corresponding to #F55422.

<img width="202" alt="Screenshot 2024-12-11 at 12 25 04" src="https://github.com/user-attachments/assets/172d51c7-62fc-45ab-b5ac-309be4395517">

### Expected Behavior

I'd expect the color to be actually red.

### Steps To Reproduce

Just write some text and then change it's color to red

### Environment

```markdown
- OS: macOS 15.1.1
- Version: 0.43.8
```

**Comment by SeiRann** [comment]
I have opened a PR about this [fix](https://github.com/anyproto/anytype-ts/pull/1112#issue-2753640864). I couldn't really build anything cause I have recently moved and my internet is not really stable and can't really see the change of color, if you can build it or test it let me know.

**Comment by alessandromrc** [comment]
> I have opened a PR about this [fix](https://github.com/anyproto/anytype-ts/pull/1112#issue-2753640864). I couldn't really build anything cause I have recently moved and my internet is not really stable and can't really see the change of color, if you can build it or test it let me know.

I have tried myself to build Anytype but always failed... Probably because I am on an Apple Silicon Mac.

**Comment by SeiRann** [comment]
I don't think so, I get like an early EOF error when I try to clone the repo, but I really think it's because I have very slow internet speeds and also unstable. You should be good, Apple devices are really good, if you have stable internet connection then it might be some other issue.

**Comment by alessandromrc** [comment]
> I don't think so, I get like an early EOF error when I try to clone the repo, but I really think it's because I have very slow internet speeds and also unstable. You should be good, Apple devices are really good, if you have stable internet connection then it might be some other issue.

Well the problem isn't the machine itself but dependencies not being compatible with ARM (If I remember when I was running "npm run update:locale" it would fail instantly because it wouldn't be able to find a version for ARM.

**Comment by SeiRann** [comment]
Interesting, did you manage to build [anytype-heart](https://github.com/anyproto/anytype-heart)?. I was also having issues with dependecies whenever I tried to run `npm install -D`, something about the patch thingy.

**Comment by SeiRann** [comment]
> > I don't think so, I get like an early EOF error when I try to clone the repo, but I really think it's because I have very slow internet speeds and also unstable. You should be good, Apple devices are really good, if you have stable internet connection then it might be some other issue.
> 
> Well the problem isn't the machine itself but dependencies not being compatible with ARM (If I remember when I was running "npm run update:locale" / "npm run dist" it would fail instantly because it wouldn't be able to find a version for ARM.
> 
> You can see #83

I guess, I don't really know stuff about ARM, as I am building this on Windows, for me is really just an internet issue. Probably changing some dependencies around could fix it, but really idk.

**Comment by alessandromrc** [comment]
This is the log that I get when trying to run update.sh
 
```
➜  anytype-ts git:(main) ./update.sh macos-latest arm
Arch: arm
Folder: build

Version: "v0.38.7"
Found asset: null
Downloading file...Done
Uncompressing... tar: Error opening archive: Unrecognized archive format
Done
Moving...
mv: rename grpc-server to build/anytypeHelper: No such file or directory
mv: rename protobuf/* to dist/lib/*: No such file or directory
mv: rename json/* to dist/lib/json/generated/*: No such file or directory
Done
```

I could probably build it from scratch but I really don't have much time to waste over this.

**Comment by alessandromrc** [comment]
@SeiRann I just built it and executed it with your change but the color seems to still be orange sadly

![Screenshot 2024-12-21 at 01 38 42](https://github.com/user-attachments/assets/866e09c9-7d20-4170-be3e-f6fb2e90269d)

**Comment by SeiRann** [comment]
I will try to make more changes. But because of my shitty internet I can't build it, and test it properly.

**Comment by alessandromrc** [comment]
> I will try to make more changes. But because of my shitty internet I can't build it, and test it properly.

I just told you the changes you should do in the PR

**Comment by ra3orblade** [comment]
Closing by design team answer here: https://github.com/anyproto/anytype-ts/pull/1112
